### ISSUE TITLE
The shared/annotations test is taking long time to run

### ISSUE THREAD

**Issue by wojciechka** [influxdata/ui#1912]
The `shared/annotations` test it taking around 7-8 minutes to run in E2E tests.

This is not caused by errors, but simply by amount of tests and time it takes to run all the tests.

Would it be possible to split it into multiple files so that they are run in parallel?

**Comment by genehynson** [comment]
The test suite has grown some more which is **awesome** but it is also causing the UI CI to exceed the 50min timeout. We should split this file into multiple files so we can run them in parallel.
